stack/indexer/rpm/build_package.sh

```diff
@@ -42,14 +43,16 @@ build_rpm() {
     # Copy the necessary files
     cp ${current_path}/builder.sh ${dockerfile_path}
 
-    # Base generation
-    if [ "${future}" == "yes" ];then
-        base_cmd+="--future "
+    if [ "${build_base}" == "yes" ];then
+        # Base generation
+        if [ "${future}" == "yes" ];then
+            base_cmd+="--future "
+        fi
+        if [ "${reference}" ];then
+            base_cmd+="--reference ${reference}"
+        fi
```

Comment:
Same, use `=` in if statements

Reply:
I think it is better to use the operator "==" since with this we keep this operator that is used in other programming languages. Any of the options is valid "=" or "=="